app/views/discovered_hosts/welcome.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for(:title, _("DiscoveredHosts")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("gears", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Discovered Hosts') %></h1>
+  <p><%= _("You don't have any visible discovered hosts. This plugin enables Foreman to do automatic bare-metal discovery of unknown nodes on the provisioning network.") %></p>
+  <p><%= link_to _('Learn more about this in the documentation.'), documentation_url("8.0/index.html", {:root_url => 'https://theforeman.org/plugins/foreman_discovery/'})%></p>
```

Comment:
Use something like:

    ForemanDiscovery::VERSION.scan(/\d+\.\d+/).first

to get the current stable version number for the URL. Docs are just about to appear there this week.